eagle-server/src/main/resources/application.conf

```diff
@@ -115,4 +115,49 @@ coordinator {
     initDelayMillis = 1000
     delayMillis = 30000
   }
+}
+
+# ---------------------------------------------
+# Eagle Authentication Configuration
+# ---------------------------------------------
+
+auth {
```

Comment:
It's better to keep reading configuration from `configuration.yml` in dropwizard stack.